test/navigationLibrary/StandardRoute/StandardRouteModel.spec.js

```diff
@@ -16,6 +16,7 @@ import { FIX_LIST_MOCK } from '../Fix/_mocks/fixMocks';
 
 import {
     STAR_LIST_MOCK,
+    STAR_WITH_SIFFIX,
```

Comment:
Is this use of S_I_FFIX intentional?

Looks genuine typo to me but of course I'm not native (nor that familiar with aviation) so it might be something I just don't know.